### DISCUSSION TITLE
MiSTer symlinks for saves/savestates/BIOS

### DISCUSSION THREAD

**Discussion by HylianPeasant** [retronas/discussions#67] [Ideas]
Hello,

Just an idea regarding the current MiSTer structure (created after installing "Samba for Mister - MiSTer FPGA CIFS config").

Currently, symlinks are created only for ROMS like shown below :
/data/retronas/mister/games/%mister_system_name%       ->          /data/retronas/roms/%retronas_vendor_name%/%retronas_system_name%

Would it be possible also to create symlinks for saves, savestates and BIOS like this ?
/data/retronas/mister/saves/                                                 ->          /data/retronas/saves/misterfpga/
/data/retronas/mister/savestates/                                         ->          /data/retronas/savestates/misterfpga/
/data/retronas/mister/BIOS/%mister_system_name%/         ->          /data/retronas/bios/misterfpga/%mister_system_name%

The idea for this would be to have the option of using the retronas as a central repository for MiSTer.
The main benefit would be to be able to use multiple MiSTer pointing to the same SMB/CIFS share in order to continue saved games (or snapshot of saved games).

Also I think it would better match the philosophy of retronas which is centralizing all the stuff by abstracting the resources part (roms, iso, saves etc).

Please let me know what you think of this idea ?

HP

**Comment by sairuk** [comment]
there is a thread discussing this and the support is in RetroNAS it is just not enabled by default

See https://github.com/danmons/retronas/discussions/60#discussioncomment-2444763

**Comment by HylianPeasant** [comment]
Ok thanks, despite my searches, didn't found that topic. Apologies.
Enabled those options on the playbook file and I confirm this work as intended.
Will report later again but this look to be mature enough to be enabled by default.
